File: game-server/src/main/java/work/fking/pangya/game/packet/handler/PapelShopPlayPacketHandler.java
package work.fking.pangya.game.packet.handler;

import io.netty.buffer.ByteBuf;
import work.fking.pangya.game.GameServer;
import work.fking.pangya.game.player.Player;
import work.fking.pangya.game.net.ClientPacketHandler;
import work.fking.pangya.game.packet.outbound.PapelShopReplies;

public class PapelShopPlayPacketHandler implements ClientPacketHandler {

    @Override
    public void handle(GameServer server, Player player, ByteBuf packet) {
        player.channel().writeAndFlush(PapelShopReplies.success());
    }
}

File: game-server/src/main/kotlin/work/fking/pangya/game/packet/handler/PapelShopPlayPacketHandler.kt
package work.fking.pangya.game.packet.handler

import io.netty.buffer.ByteBuf
import work.fking.pangya.game.GameServer
import work.fking.pangya.game.net.ClientPacketHandler
import work.fking.pangya.game.packet.outbound.PapelShopReplies
import work.fking.pangya.game.player.Player

class PapelShopPlayPacketHandler : ClientPacketHandler {
    override fun handle(server: GameServer, player: Player, packet: ByteBuf) {
        player.writeAndFlush(PapelShopReplies.success())
    }
}
